spec/src/main/asciidoc/rationale.asciidoc

```diff
@@ -0,0 +1,80 @@
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+[[reactivemessagingrationale]]
+== Rationale
+
+State-of-the-art systems must be able to adapt themselves to emerging needs and requirements, such as market change and user expectations but also fluctuating load and inevitable failures.
+Leading-edge applications are capable of dynamic and adaptive capabilities aiming to provide _responsive_ system.
+While microservices aim to offer this agility, HTTP-based connecting tissue tends to fail to provide the required runtime adaptations, especially when facing failures.
+
+Asynchronous communication allows temporal decoupling of services in a microservice based architecture. 
+This temporal decoupling implies that communication is allowed to occur regardless of when the parties involved in the communication are running, whether they are loaded or overloaded, and whether they are successfully processing messages or failing.
+
+In contrast, synchronous communication couples services together, binding their uptime, failure, and handling of the load to each other. 
+In a chain of synchronous interactions, the entire conversation can only be successful if all parties in the chain are responsive - if they are all running, processing messages successfully, and not overloaded. 
+If just one party has a problem, all effectively exhibit the same problem.
+Therefore, systems of microservices relying on HTTP tend to be _fragile_, and failures limit their availability.
+Indeed, in a microservice-based architecture, temporal coupling results in a fragile system, with resilience and scaling properties that are worse than a monolith, hence, it is essential for microservice based architectures to embrace asynchronous communication as much as possible. 
+
+The role of the MicroProfile Reactive Messaging specification is to deliver a way to build systems of microservices promoting both location transparency and temporal decoupling, enforcing asynchronous communication between the different parts of the system.
+
+=== Reactive Systems
+
+https://www.reactivemanifesto.org/[Reactive Systems] provide an architecture style to deliver _responsive_ systems. 
+By infusing asynchronous messaging passing at the core of the system, applications enforcing the reactive system's characteristics are inherently elastic and resilient.
+
+image::reactive-system.png[Reactive System Characteristics, 400, 400]
+
+Microservices part of a reactive systems interacts using _messages_.
+The location and temporal decoupling, promoted by this interaction mechanism, exhibits numerous benefits such as:
+
+* Better failure management as the temporal decoupling would avoid message-loss and avoid cascading failures.
+* Improved elasticity as under fluctuating load the system can decide to scale up and down some of the microservices.
+* The ability to introduce new feature quickly as every component receives and publishes messages.
```

Comment:
The ability to introduce new features more easily as components are more loosely coupled by receiving and publishing messages.